### PR TITLE
Create GEMDigis and make some cfg file in Cmssw11

### PR DESCRIPTION
**Pull request by SeungJunLee0**
If you have any question or find something wrong. 
please tell me .
 Here is my mail address seungjun.lee@cern.ch, seungjunlee15@gmail.com 
Thank you

### CODE REVIEW

File: python/ispy_GEMs_cfg.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+#from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
+from Configuration.AlCa.GlobalTag import GlobalTag                                ##
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2021_realistic', '')## 
+
+import FWCore.Utilities.FileUtils as FileUtils
+
+process.source = cms.Source(
+    'PoolSource',
+    fileNames = cms.untracked.vstring(
```
[the remark this answers is not in the document]
Comment (tpmccauley) [reply python/ispy_GEMs_cfg.py:30]: Resolved since I now have ig files for testing

File: src/ISpyGEMSegment.cc [hunk 2]
```diff
@@ -123,9 +122,9 @@ ISpyGEMSegment::analyze (const edm::Event& event, const edm::EventSetup& eventSe
       // along y
       float halfLength = det->surface().bounds().length() / 2.0;
 
-      //std::cout<<"px, py, pz: "<< pos.x() <<" "<< pos.y() <<" "<< pos.z() <<std::endl;
-      //std::cout<<"dx, dy, dz: "<< dir.x() <<" "<< dir.y() <<" "<< dir.z() <<std::endl;
-      //std::cout<<"t, w, l: "<< thickness <<" "<< width <<" "<< length <<std::endl;
+      //Std::cout<<"px, py, pz: "<< pos.x() <<" "<< pos.y() <<" "<< pos.z() <<std::endl;
```
Comment (tpmccauley) [inline src/ISpyGEMSegment.cc:125]: Please remove printout here and below

File: src/ISpyMuon.cc [hunk 3]
```diff
@@ -286,10 +286,12 @@ ISpyMuon::addChambers(reco::MuonCollection::const_iterator it)
     if ( dit->detector() == MuonSubdetId::GEM )
     {
       geomDet = gemGeometry_->idToDet((*dit).id);
+	  std:: cout << "ingme" <<std::endl;
```
Comment (tpmccauley) [inline src/ISpyMuon.cc:289]: please rm printout

File: src/ISpyMuon.cc [hunk 4]
```diff
@@ -286,10 +286,12 @@ ISpyMuon::addChambers(reco::MuonCollection::const_iterator it)
...
     }
     else if ( dit->detector() == MuonSubdetId::CSC )
     {
       geomDet = cscGeometry_->idToDet((*dit).id);
+	  std:: cout << "icsc" <<std::endl;
```
Comment (tpmccauley) [inline src/ISpyMuon.cc:294]: please rm printout

File: python/ispy_GEMs_cfg.py [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    #'root://cmsxrootd.fnal.gov//store/data/Run2018D/DoubleMuon/AOD/PromptReco-v2/000/324/998/00000/AF519538-7FE5-4A4B-BD66-6FE4900CB5C6.root'
+    # '/store/data/Commissioning2021/Cosmics/RAW-RECO/CosmicTP-PromptReco-v1/000/341/169/00000/095946f3-e67a-4b4f-a098-3432372b6d0e.root'
+   #  '/store/group/alca_muonalign/run3MC/step2_501.root'
+     'root://cms-xrdr.private.lo:2094//xrd/store/user/seungjun/ISPY/display_out_full.root'
```
Comment (tpmccauley) [inline python/ispy_GEMs_cfg.py:34]: Perhaps just keep the two files here that you used to produce the ig files and rm the other ones?

File: src/ISpyMuon.cc (hunk 4 above)
Comment (SeungJunLee0) [reply src/ISpyMuon.cc]: i am sorry... i remove it

File: python/ispy_GEMs_cfg.py (hunk 5 above)
Comment (SeungJunLee0) [reply python/ispy_GEMs_cfg.py]: i keep one and remove the others

File: src/ISpyMuon.cc (hunk 4 above)
Comment (SeungJunLee0) [reply src/ISpyMuon.cc]: i solve it

File: src/ISpyMuon.cc (hunk 3 above)
Comment (SeungJunLee0) [reply src/ISpyMuon.cc]: i solved it

File: src/ISpyGEMSegment.cc (hunk 2 above)
Comment (SeungJunLee0) [reply src/ISpyGEMSegment.cc]: i solved it

**Comment by tpmccauley** [comment]
Thanks @SeungJunLee0

**Comment by SeungJunLee0** [comment]
Thanks!!! @tpmccauley

**Pull request merged**